boards/riscv/tlsr9518adk80d/tlsr9518adk80d_defconfig

```diff
@@ -7,7 +7,7 @@ CONFIG_BOARD_TLSR9518ADK80D=y
 CONFIG_PLIC=y
 CONFIG_RISCV_MACHINE_TIMER=y
 CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
-CONFIG_PINMUX=y
+CONFIG_PINCTRL=y
```

Comment:
consider defaulting at SoC level, since pinctrl is in practice always required. See e.g. https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/arm/gigadevice/Kconfig.defconfig#L8